staging/src/k8s.io/client-go/util/workqueue/delaying_queue.go

```diff
@@ -208,8 +302,6 @@ func (q *delayingType) AddAfter(item interface{}, duration time.Duration) {
 		return
 	}
 
-	q.metrics.retry()
-
 	// immediately add things with no delay
 	if duration <= 0 {
 		q.Add(item)
```

Comment:
the no delay Add was mesuring retries before (`"Total number of retries handled by workqueue"`), although I am not sure if there is benefit of preverving this behaviour

Reply:
Gonna leave as-is as there isn't really an elegant solution and a whole new set of metrics may be appropriate but should be a follow up PR.